src/main/scala/pt/tecnico/dsi/openstack/keystone/services/Domains.scala

```diff
@@ -28,18 +27,27 @@ final class Domains[F[_]: Sync: Client](baseUri: Uri, authToken: Header)
     )))
 
   /**
-    * Get detailed information about the domain specified by name.
+    * Get detailed information about the domain specified by name, assuming it exists.
     *
     * @param name the domain name
-    * @return the domain matching the name.
+    * @return the domain matching the name. If none exists F will contain an error.
```

Comment:
What do you mean F will contain an error? Shouldn't we wrap the call in an `Option`?